graphwalker-studio/pom.xml

```diff
@@ -75,6 +75,25 @@
           <addResources>true</addResources>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>pl.project13.maven</groupId>
+        <artifactId>git-commit-id-plugin</artifactId>
+        <version>2.2.3</version>
```

Comment:
I wonder if this shouldn't be in the root instead, giving all the different jar files a git.properties file

Reply:
or maybe it's unnecessary, feel free to disregard this comment =)

Reply:
Good point. Should be written once, shared by all.
I'll update the PR